README.md

```diff
@@ -199,16 +199,20 @@ please DO NOT file a public issue, instead send your report privately to
 `security@coredns.io`. Security reports are greatly appreciated and we
 will publicly thank you for it.
 
+CoreDNS Team may disclose privately the security issue in order to organize the patch and its release.
 
-### Coordination of security releases with Kubernetes project
+If you lead a project that includes CoreDNS and need organizing controlled security releases when CoreDNS triggers a security release, 
+contact us at `security@coredns.io` in order to be added to our "Collaborating Projects" and have access to early security release information.
 
-As CoreDNS is now a feature part of Kubernetes, we are organizing our security release process 
-in a way that is compatible with [Kubernetes's security release process](https://github.com/kubernetes/sig-elease/blob/master/security-release-process-documentation/security-release-process.md)
+### Coordination of security releases with Collaborating Projects
 
-1. CoreDNS Team may disclose privately the security issue in order to organize the patch and its release.
-1. CoreDNS Team will inform Kubernete's project using kubernetes-security@googlegroups.com as early as possible and coordinate 
-with Kubernete's Team the upcoming security release timeline in such a way that Kubernetes 
+CoreDNS manage a list of "Collaborating Projects" that are assessed CoreDNS consumers and reliable secure projects. 
+That list aims to coordinate the security release in a controlled manner: 
+ 
```

Comment:
In order to be transparent and for each project to be able to validate they are registered, should we create a public document CollaboratingProjects.md  with the public information regarding process and agreement with each project ?